staff/victor-abadia/horroy/scripts/horroy.test.js

```diff
@@ -0,0 +1,56 @@
+// TODO do it nice!
+suite('horroy', function () {
+
+    describe('Push success cases', function () {
+
+        it('should add an element to the horroy', function () {
+
+            var horroy = new Horroy();
+
+            horroy.push(7) // [7]
+
+            expect(horroy[0] == 7, 'not expected result')
+            expect(horroy.length === 1, 'not expect result')
+
+        });
+
+    });
+
+    describe('Push error cases', function () {
+
+        it('should fail if element is undefined', function () {
+
+            var horroy = new Horroy();
+            var error;
+
+            try {
+                horroy.push()
+            } catch (err) {
+                error = err
+            }
+
+            expect(error, 'was expect an error')
+
+        });
+
+    });
+
+    describe('Pop success cases', function () {
+
+        it('should delete the last element in the horroy', function () {
+
+            var horroy = new Horroy('a', 'b', 'c');
+
+            horroy.pop() // horroy = Horroy {0: "a", 1: "b", length: 2}
+
+            var expect = new Horroy(a,b)
```

Comment:
Hey, I don't thing that test it's working! You are not passing a and b as string, instead as variables. Also, the expect is not constructed well